### PR TITLE
Hunter v2.1.2

### PR DESCRIPTION
**Pull request by jowr**
I tried to follow the [online guide](https://docs.hunter.sh/en/latest/creating-new/patch.html#patch-sources), but I am not sure I succeeded. Would you mind having a look?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.0.0)
+option(HUNTER_ENABLED "Enable Hunter package manager" NO)
```
Comment (ruslo) [inline CMakeLists.txt]: This is not needed, this project is not using Hunter to manage dependencies as far as I can see.

File: CMakeLists.txt [hunk 2]
```diff
@@ -130,6 +131,57 @@ if(IF97_CMAKE_MODULE)
     )
     set(include_install_dir "include")
     install(FILES IF97.h DESTINATION "${include_install_dir}")
+
+    if(HUNTER_ENABLED)
+        set(config_install_dir "lib/cmake/${PROJECT_NAME}")
```
Comment (ruslo) [inline CMakeLists.txt]: To be precise it's not a "hunterization". This is a common CMake practice: https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html

`if(HUNTER_ENABLED)` should be removed. The rest of the code should be submitted upstream.

**Comment by jowr** [comment]
OK, thanks for your patience with me. This implicitly means that this project should not need to have any hunter-specific code, right? Should it be removed from the hunter-packages repository?

**Comment by ruslo** [comment]
> This implicitly means that this project should not need to have any hunter-specific code, right?

Yes.

> Should it be removed from the hunter-packages repository?

When this change will be available upstream - yes. For now it can be used for hosting patched version before official release.

Note also that even "hunterized" code can be send upstream without affecting functionality (like you've used `option(HUNTER_ENABLED "..." NO)`). As [documentation states](https://docs.hunter.sh/en/latest/creating-new/patch.html) "the final goal of this organization is to have no forks of packages".

**Comment by jowr** [comment]
Thanks for the clarification. Guess we can delete this repository since I am one of the upstream authors. I'll make a new release in the main repository.

**Pull request closed without merging**